CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ When updating the changelog, remember to be very clear about what behavior has c
 and what APIs have changed, if applicable.
 
 ## [Unreleased]
+Add equals and hashCode methods to CollectionResult, GetResult, UpdateResponse and UpdateEntityResponse.
```

Comment:
nit: preface with `-` bullet point, add code backticks "`" around the class names.